build/build-lifetime.sh

```diff
@@ -32,7 +32,7 @@ cmake -G "Ninja" ../llvm-project/llvm \
     -DCMAKE_BUILD_TYPE:STRING=Release \
     -DLLVM_ENABLE_PROJECTS=clang \
     -DCMAKE_INSTALL_PREFIX:PATH=/root/staging \
-    -DLLVM_BINUTILS_INCDIR:PATH=/opt/compiler-explorer/gcc-7.3.0/lib/gcc/x86_64-linux-gnu/7.3.0/plugin/include/
+    -DLLVM_BINUTILS_INCDIR:PATH=/opt/compiler-explorer/gcc-${BINUTILS_GCC_VERSION}/lib/gcc/x86_64-linux-gnu/${BINUTILS_GCC_VERSION}/plugin/include/
```

Comment:
BINUTILS_GCC_VERSION doesn't appear to be defined in this script anywhere

Reply:
Committed too fast, fixed